app/presenters/publishing_api_presenters/links_presenter.rb

```diff
@@ -42,6 +46,33 @@ def policy_area_ids
       (item.try(:topics) || []).map(&:content_id)
     end
 
+    def related_guides_ids
```

Comment:
Do you know if there are more formats using related guides & friends? If not it might be better to move this to `PublishingApiPresenters::DetailedGuide`.